src/web/app/pages-student/student-home-page/student-home-page.component.ts

```diff
@@ -11,14 +103,102 @@ import { ActivatedRoute } from '@angular/router';
 })
 export class StudentHomePageComponent implements OnInit {
 
+  // Tooltip messages
+  studentCourseDetails: string = 'View and edit information regarding your team';
+
+  studentFeedbackSessionStatusPublished: string =
+      'The responses for the session have been published and can now be viewed.';
+  studentFeedbackSessionStatusNotPublished: string =
+      'The responses for the session have not yet been published and cannot be viewed.';
+  studentFeedbackSessionStatusAwaiting: string =
+      'The session is not open for submission at this time. It is expected to open later.';
+  studentFeedbackSessionStatusPending: string = 'The feedback session is yet to be completed by you.';
+  studentFeedbackSessionStatusSubmitted: string = 'You have submitted your feedback for this session.';
+  studentFeedbackSessionStatusClosed: string = ' The session is now closed for submissions.';
+
+  feedbackSessionEditSubmittedResponse: string = 'Edit submitted feedback';
+  feedbackSessionViewSubmittedResponse: string = 'View submitted feedback';
+  feedbackSessionSubmit: string = 'Start submitting feedback';
+  feedbackSessionResults: string = 'View the submitted responses for this feedback session';
```

Comment:
All the above don't need to be variables if they are only used once. Just write them in the HTML files that use them.